client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -130,7 +137,11 @@ export default memo( function SitesTableRow( { site }: SiteTableRowProps ) {
 							<JetpackLogo size={ 16 } />
 						</SitePlanIcon>
 					) }
-					{ site.plan?.product_name_short }
+					{ isExpiredAtomic( site ) ? (
```

Comment:
Should we show this for all expired sites, not just Atomic?

Reply:
Not sure about that. This is what I understood from the description of the issue.
Maybe @danielbachhuber can help here?

Reply:
@kozer Hi, original commenter here 😁

I think we can show this nag for all expired sites, not just expired Atomic sites. Given we'd like users to renew, I think it makes sense to show it to more applicable users.

Reply:
Oooopsie @danielbachhuber :D. So the first "is done" doesn't apply? 
It states:
"When an Atomic site's plan is expired, but the site isn't reverted, we display a notice on the Sites Management Page."

Reply:
> Oooopsie @danielbachhuber :D . So the first "is done" doesn't apply? It states: "When an Atomic site's plan is expired, but the site isn't reverted, we display a notice on the Sites Management Page."

It broadens the scope a bit: _When an ~Atomic~ site's plan is expired, ~but the site isn't reverted,~ we display a notice on the Sites Management Page._

Reply:
Great! thanks for the explanation! Will make the change!

Reply:
@kozer Yep! Sorry I wasn't clear enough in the first place.